scripts/base/frameworks/spicy/main.zeek

```diff
@@ -0,0 +1,15 @@
+@load base/frameworks/notice
+
+module Spicy;
+
+export {
+    redef enum Notice::Type += { Spicy_Max_File_Depth_Exceeded };
+}
+
+event max_file_depth_exceeded(f: fa_file, args: Files::AnalyzerArgs, limit: count)
+    {
+    NOTICE([
```

Comment:
I understand this is what the spicy-plugin does today, but wonder if weird.log might have been better (and/or propagating it to analyzer.log).

If users trigger this regularly, they need to write log policy hooks to tame the notice.log. notice.log is also a very popular log and avoiding noise probably a good idea (as in contrast to weird.log).

Not sure worth changing now though, but thought I'd leave the thoughts.

Reply:
Yeah, sounds like a separate discussion, can see it both ways. I'll leave that for some other time.